src/main/java/de/cerus/jdasc/JDASlashCommands.java

```diff
@@ -534,17 +534,17 @@ public static void handleInteraction(final Interaction interaction) {
     }
 
 
+    //TODO: figure out whats null
```

Comment:
Would be great if you could fix that in a future pr

Reply:
Yeah ill look into  it :D